Gemfile

```diff
@@ -99,11 +99,13 @@ gem 'puma'
 gem 'therubyracer', platform: :ruby # C Ruby (MRI) or Rubinius, but NOT Windows
 gem 'sass-rails'
 gem 'uglifier'
-# We can't upgrade because not compatible to jquery >= 1.9.
-# To do that, we need fix the rails.js
-gem 'jquery-rails', '~> 2.1.4'
+gem 'jquery-rails', '~> 4.1'
 gem 'pjax_rails'
 gem 'underscore-rails'
 
+gem 'bootstrap-sass', '~> 3.3.6'
+gem 'coffee-rails'
```

Comment:
There doesn't appear to be any coffeescript in the project, do we need this gem then? (Awesome work btw, I really hope this gets merged! :+1: )